dbms/tests/queries/0_stateless/00725_ipv4_ipv6_domains.sql

```diff
@@ -0,0 +1,59 @@
+DROP TABLE IF EXISTS ipv4_test;
```

Comment:
In stateless tests tables should be created in `test` database.

Reply:
fixed